core/touch_gesture.ts

```diff
@@ -202,10 +197,10 @@ export class TouchGesture extends Gesture {
   }
 
   /**
-   * Handle a touch start or pointer down event and keep track of current
+   * Handle a pointerdown event and keep track of current
    * pointers.
    *
-   * @param e A touch start, or pointer down event.
+   * @param e A pointerdown event.
    * @internal
    */
   handleTouchStart(e: PointerEvent) {
```

Comment:
Can `handleTouchStart` be absorbed into the `handleStart` method(s)?

Reply:
In terms of inlining it.

Reply:
Given that it's called from multiple places I think it's probably preferable to leave as-is?

Reply:
Sgtm